packages/checklist-block/src/components/Checkbox.tsx

```diff
@@ -53,6 +53,7 @@ export const Checkbox: FC<CheckboxProps> = ({
     };
 
     return (
+        // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions
```

Comment:
Do you think it makes sense to add an onKeyDown listener for the "Enter" key that also calls `handleLabelClick` ?

Reply:
Won't tackle as its an unreleased block and needs some work anyway before go live